libraries/c_sdk/aws/defender/test/system/aws_iot_tests_defender_system.c

```diff
@@ -74,7 +74,7 @@
 #define MAX_ADDRESS_LENGTH                   25
 
 /* Use a big number to represent no event happened in defender. */
-#define NO_EVENT                             10000
+#define NO_EVENT                             ( 1u << ( 8 * sizeof( AwsIotDefenderEventType_t ) - 1 ) )
```

Comment:
Resolves the following `-Woverflow` warning:
```
/home/ubuntu/Repos/amazon-freertos/libraries/c_sdk/aws/defender/test/system/aws_iot_tests_defender_system.c:77:46: warning: unsigned conversion from 'int' to 'enum <anonymous>' changes value from '10000' to '16' [-Woverflow]
   77 | #define NO_EVENT                             10000
      |                                              ^~~~~
/home/ubuntu/Repos/amazon-freertos/libraries/c_sdk/aws/defender/test/system/aws_iot_tests_defender_system.c:568:22: note: in expansion of macro 'NO_EVENT'
  568 |         .eventType = NO_EVENT
      |                      ^~~~~~~~
```

Reply:
What is the logic behind this expression?

Reply:
The comments mention about using a large number for the test, so this logic stores the largest value for the compiler-chosen bit-width of the `AwsIotDefenderEventType_t ` type

Reply:
For simpler readability, have removed the special expression

Reply:
Why not make the `AwsIotDefenderEventType_t` Enum big enough to hold this (10000) value? 32-bit values can be processed much more easily I believe (since we don't have 8/16-bit processors).
Are there any downsides of doing that?